app/frontend/src/common/assets/js/leaflet-areaselect.js

```diff
@@ -0,0 +1,184 @@
+L.AreaSelect = L.Class.extend({
```

Comment:
Please attribute this file to the original author and state the license terms (MIT license).
A comment saying Copyright [author's name] and link to the original repo, and "licensed under the MIT license" should suffice.

Reply:
Done